### ISSUE TITLE
is there any way to print all the findings instead of 50?

### ISSUE THREAD

**Issue by dv000-7** [ankane/pdscan#20]
hi,

is there any way to print all the findings instead of 50?
i using pdscan to scan the databases but it is showing only 50 findings not other records

       pdscan = [
            "pdscan",
            "--only", "card",
            f"{self.database}://{self.username}:{self.password}@{self.host}:{self.port}/{self.db_name}",
            "--format", "ndjson",  
            "--show-data",
            "--show-all"
         ] 
     any suggestions, advice will be helpful...!! 
   Need  help  @here
   thanks

**Comment by vinaykmatam** [comment]
yes, It will be really helpfull if was show full data instead of limited on screen @ankane can we have this feature?
